package.json

```diff
@@ -1,15 +1,10 @@
 {
   "private": true,
   "scripts": {
-    "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "eslint src && pulp build -- --censor-lib --strict",
-    "test": "pulp test"
+    "build": "eslint src && spago build --purs-args '--censor-lib --strict'",
+    "test": "spago test --no-install"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "pulp": "^12.2.0",
-    "purescript-psa": "^0.6.0",
```

Comment:
Should we be removing `purescript-psa` here?

Reply:
Good point; it's not normally included because we normally don't even have a package.json file, and Spago is assumed as a requirement for all libraries. But in this case you're already going to `npm i` and then use a build command that explicitly relies on psa, so we may as well include it.